site/_posts/2019-09-20-Velero-v1-1-Stateful-Backup-vSphere.md

```diff
@@ -0,0 +1,460 @@
+---
+title: Velero v1.1 backing up and restoring Stateful apps on vSphere and PKS (Pivotal Container Service)
+excerpt: This post demonstrates how Velero can be used on PKS, the Pivotal Container Service which deploys Kubernetes clusters on vSphere.
```

Comment:
Any chance you could have an image to use for this post, so we don't default to the Velero logo?

Reply:
Added Cassandra logo as that is the app I am backing up and restoring.